docs/source/AdministratorGuide/Configuration/ConfReference/Operations/Pilots/index.rst

```diff
@@ -13,6 +13,12 @@ LCGBundleVersion    which lcgBundle version to install with the pilot.        No
                     Be careful: if defined, this version will overwrite
                     any possible version defined in the releases.cfg file
 ------------------  --------------------------------------------------------  -----------------------------------------------------------------------------------------------
+DIRACOS             If set, it will use the --dirac-os options to install     False
+                    DIRAC within the pilot
+                    Be careful: if defined, this version will overwrite
+                    any possible version of LCGbundle
```

Comment:
The version of DIRACOS is taken from the releases.cfg? Or is there some way to overwrite the version used?

Reply:
Yes it is. Right now the pilot 3 does not have an option to overwrite it. Pilot2 has no DIRACOS option at all.